data-raw/process_data_sets_kara.R

```diff
@@ -0,0 +1,35 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#reading in csv
+nba_elo <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo.csv")
+nba_elo_latest <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo_latest.csv")
+
+rownames(nba_elo) <- NULL
+rownames(nba_elo_latest) <- NULL
+
+#tidying and taming data for nba_elo
+nba_elo <- nba_elo %>% clean_names()
+
+nba_elo <- nba_elo %>%
+  mutate_if(is.logical, as.numeric)
+
+nba_elo$neutral <- as.logical(nba_elo$neutral)
+
+
+#tidying and taming data for nba_elo_latest
+nba_elo_latest <- nba_elo_latest %>% clean_names()
+
+nba_elo_latest <- nba_elo_latest %>%
+  mutate_if(is.logical, as.numeric)
+
+nba_elo_latest$neutral <- as.logical(nba_elo_latest$neutral)
```

Comment:
Could you use:
* `mutate()` instead?
* Combine all `%>%` into a single pipe chain? 

Kind of like in this section of the [tidyverse style guide](https://style.tidyverse.org/pipes.html#whitespace)